driver-core/src/main/com/mongodb/internal/validator/MappedFieldNameValidator.java

```diff
@@ -56,4 +56,14 @@ public FieldNameValidator getValidatorForField(final String fieldName) {
             return defaultValidator;
         }
     }
+
+    @Override
+    public void start() {
+        defaultValidator.start();
```

Comment:
I'm not sure these overrides are necessary, or correct.  Are there tests that prove it either way?

Reply:
Let me know on this one.

Reply:
I have debugged this, and there are no tests that prove it either way. My thinking here was to emulate how `defaultValidator` was used in other methods, such as `validate()`. I have removed the `start()` and `end()` methods from this class.